src/plugins/usage_collection/public/services/create_reporter.ts

```diff
@@ -27,8 +28,8 @@ export function createReporter(config: AnalyicsReporterConfig): Reporter {
         body: JSON.stringify({ report }),
         asSystemRequest: true,
       });
-
-      if (response.status !== 'ok') {
+      const okStatus: UiCounters.v1.UiCountersResponseOk = response.status;
+      if (response.status !== okStatus) {
```

Comment:
The responses are hard-coded, depending on whether `fetch` throws or not, so we declare separate types for each outcome.